lib/decimal.ex

```diff
@@ -2,8 +2,8 @@ defmodule Decimal do
   @moduledoc """
   Decimal arithmetic on arbitrary precision floating-point numbers.
 
-  A number is represented by a signed coefficient and exponent such that: `sign
-  * coefficient * 10^exponent`. All numbers are represented and calculated
+  A number is represented by a signed coefficient and exponent such that: ```sign
+  * coefficient * 10^exponent```. All numbers are represented and calculated
```

Comment:
@ericmj It is probably a bug in the parser. I'll open a bug report there.
![decimal_ _decimal_v1_1_1-dev](https://cloud.githubusercontent.com/assets/1635127/11283033/b37bc128-8ee9-11e5-8217-df2b5a90823e.png)
I was able to update ex_doc and it rendered the page with the same error.